python/lsst/ts/FiberSpectrograph/avsFiberSpectrograph.py

```diff
@@ -292,6 +292,40 @@ def decode(value):
                                     config=config if full else None)
         return status
 
+    def check_expose_ok(self, duration):
+        """Check that an expose command would be allowed to succeed.
+
+        Checks that parameters are in a valid range, and that no exposures are
+        currently ongoing, returning a message describing the failure state if
+        there would be one, or `None` otherwise.
+
+        Parameters
+        ----------
+        duration : `float`
+            Integration time of the exposure in seconds.
+
+        Returns
+        -------
+        msg : `str` or `None`
+            The string describing why ``expose`` would fail, or None if an
+            it should not.
+
+        Notes
+        -----
+        The ``Parameters`` of this method should match that of ``expose``, as
+        its purpose is to check the validity of those parameters.
+        """
+        if not self._expose_task.done():
+            return "Cannot start new exposure until current exposure finishes."
+
+        # minimum and maximum time in seconds
+        min_t = 0.000002
+        max_t = 600
```

Comment:
I (still) suggest making these public constants so you can use them in unit tests.

Reply:
By that, do you mean testing `duration = min_t - epsilon` and `duration = max_t + epsilon`?

Reply:
Yes